Sources/HAP/Base/Generated.swift

```diff
@@ -2,13 +2,13 @@
 // framework definitions. Don't make changes to this file directly.
 // Update this file using the `hap-update` tool.
 //
-// Generated on:              26 December 2020
+// Generated on:              3 January 2021
 // HomeKit framework version: 858
 // macOS:                     Version 11.1 (Build 20C69)
 
 import Foundation
 
-public enum AccessoryType: String, Codable {
+public enum AccessoryType: String, Codable, CustomStringConvertible {
```

Comment:
Can you move the `CustomStringConvertible` to an extension, similar to what you've done with `ServiceType`?

Reply:
OK done